snappy_pipeline/workflows/ngs_mapping/__init__.py

```diff
@@ -1010,6 +1042,9 @@ class BamCollectDocStepPart(ReportGetResultFilesMixin, BaseStepPart):
     #: Step name
     name = "bam_collect_doc"
 
+    #: Run for "dna" and "dna_long" only
```

Comment:
why is bam-collection not run for RNA?

Reply:
`bam-collection` is based on the target coverage output, as far as I can tell. This is a new addition from Manuel